### PR TITLE
chore: table config popup in Mantine

### PR DESCRIPTION
**Pull request by magnew**
Closes: #6258 

### Description:

Moves the table config popup, tabs and checkboxes to Mantine. 

### Criteria
- [ ] use mantine popover
- [ ] use mantine tabs for general / conditional formatting (pie charts config as a reference)
- [ ] use mantine form elements for "options"

### CODE REVIEW

File: packages/frontend/src/components/VisualizationConfigs/TableConfigPanel/GeneralSettings.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import { Checkbox, FormGroup } from '@blueprintjs/core';
-import { Box, Title, Tooltip } from '@mantine/core';
+import { FormGroup } from '@blueprintjs/core';
```
Comment (IrakliJani) [inline packages/frontend/src/components/VisualizationConfigs/TableConfigPanel/GeneralSettings.tsx:1]: do we use FormGroup anywhere?

Comment (magnew) [reply packages/frontend/src/components/VisualizationConfigs/TableConfigPanel/GeneralSettings.tsx:1]: The drag and drop parts are still blueprint. Now that you did the z-index thing I think I can convert them too.

Comment (magnew) [reply packages/frontend/src/components/VisualizationConfigs/TableConfigPanel/GeneralSettings.tsx:1]: I looked a little more closely. It's still wrapping the column config thing that is in Blueprint. I'm planning to port that in the next patch.

Comment (IrakliJani) [reply packages/frontend/src/components/VisualizationConfigs/TableConfigPanel/GeneralSettings.tsx:1]: perfect

**Pull request merged**